modules/cache/logic/cacheLogic.js

```diff
@@ -17,8 +17,7 @@ const logger = require('../../../utils/logger')(module);
 
 const CacheLogic = {
   init() {
-    // INIT ONCE
-    cache.setKeepHot(CacheLogic.keepHotFunction);
+    CacheLogic.keepHot(CacheLogic.keepHotFunction);
   },
 
   async keepHotFunction() {
```

Comment:
why pass it as an argument? lets hardcode it or at least move it out of the init function